cobalt/h5vcc/h5vcc_system.cc

```diff
@@ -22,7 +22,6 @@
 #include "cobalt/web/environment_settings_helper.h"
 #include "cobalt_build_id.h"  // NOLINT(build/include_subdir)
 #include "starboard/common/system_property.h"
-#include "starboard/extension/ifa.h"
```

Comment:
The rest of the IFA change below seems correct, but i think you still need to keep the header included to have `StarboardExtensionIfaApi` in scope, no ?